lib/charms/pgbouncer_k8s/v0/pgb.py

```diff
@@ -38,7 +38,7 @@
 LIBAPI = 0
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 5
+LIBPATCH = 6
```

Comment:
Heads up, the charmhub's libpatch version is only 3 and charmcraft will complain that the version isn't monotonous.

Reply:
Good catch! I manually updated the version on charmhub and created a [PR](https://github.com/canonical/pgbouncer-k8s-operator/pull/41) to automatically update the library when a new version is merged on main.

Reply:
Commented the mentioned above PR.